src/translations/en.json

```diff
@@ -1372,6 +1372,7 @@
             "stop_ignore": "Stop ignoring"
           },
           "config_entry": {
+            "device_info": "Device info",
```

Comment:
This should be in the device section, not in `config_entry` (I know some are at the wrong place due to changes in the past)

Reply:
I thought it made sense to keep them in the same place

Reply:
We should move them at some point, so let's not make that harder by adding new ones at the wrong place :-)